catalog/plugin/catalog-plugin-metacardbackup/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/MetacardBackupPlugin.java

```diff
@@ -261,4 +285,24 @@ private MetacardTransformer lookupTransformerReference() {
         }
         return null;
     }
+
+    private boolean shouldBackupMetacard(Metacard metacard) {
+        if (backupInvalidMetacards) {
```

Comment:
it looks like backupInvalidMetacards could be null because the setter doesn't check for null inputs.